src/app/pages/swap/components/swap-selected-asset-from.tsx

```diff
@@ -18,14 +18,12 @@ const maxAvailableCaption = 'Max available in your balance';
 const maxAvailableTooltip =
   'Amount of funds that is immediately available for use, after taking into account any pending transactions or holds placed on your account by the protocol.';
 
-const sendAnyValue = 'Send any value';
```

Comment:
Why was this changed? Can you provide a video or screenshot of the UI and tag @mica000 to approve the design change?

Reply:
Sry, this was not intentional. Will revert.